### DISCUSSION TITLE
How to specify the item within the node that allows you to select the node?

### DISCUSSION THREAD

**Discussion by alexandreespejo** [xyflow/xyflow#3451] [Q&A]
I'm struggling to select the node only at a specific point, I see in the documentation that there is a "custom-drag-handle"
There is a class like "custom-select-handle" ? I can't find anything like this

**Comment by bcakmakoglu** [comment]
There's no "custom select handle" - usually any spot on a node is good enough to select it but if you need further control you can always just use `setNodes` (which is accessible from the [react flow instance,](https://reactflow.dev/docs/api/react-flow-instance/) for example provided by `useReactFlow`)

```ts
setNodes(nodes => nodes.map(node => {
	if (node.id === id) {
		return {
			...node,
			selected: true
		}
	}

	return node
})
```
